alex/components/dm/ontology.py

```diff
@@ -66,25 +66,27 @@ def slots_system_selects(self):
         return [slot for slot in self.ontology['slots'] if 'system_selects' in self.ontology['slot_attributes'][slot]]
 
     @lru_cache(maxsize=1000)
-    def last_talked_about(self, dat, name, value):
-        """Returns a list of slots and values that should be used to for tracking about what was talked about recently,
+    def last_talked_about(self, da_type, name, value):
+        """Returns a list of slots and values that should be used to for
```

Comment:
Can you remove reformat the comments so that full 120 characters per line are used?